.github/workflows/docker-image-manual.yml

```
@@ -27,14 +46,74 @@ jobs:
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            ${{ env.DOCKERHUB_REPO }}
+          tags: ${{ steps.extractGitBranch.outputs.branchName }}
```

actionlint (1.7.4)

property "extractgitbranch" is not defined in object type {}

(expression)